lxqt-config-appearance/lxqtthemeconfig.ui

```diff
@@ -51,6 +51,13 @@
      </column>
     </widget>
    </item>
+   <item>
+    <widget class="QCheckBox" name="wallpaperOverride">
+     <property name="text">
+      <string>Override independently set wallper</string>
```

Comment:
@jeremywh7  Most of these phrases are good. Your chosen phrase would be better in this form: "Allow theme to *set* the wallpaper". However @palinek intends to emphasise overriding.